### ISSUE TITLE
Support custom `fetch` function similar to `axiosInstance` option

### ISSUE THREAD

**Issue by smblee** [ecyrbe/zodios#392]
I am trying to use this library for SvelteKit, which provides a special `fetch` function for SSR context (https://kit.svelte.dev/docs/web-standards#fetch-apis). Is it possible to support a `fetch` function similar to `axiosInstance` maybe as an option of `pluginFetch`?

I envision my APIs to look like...

```javascript
export const PersonApi = ({ fetchFn, headers }) => {
  const client = new Zodios(PersonEndpoints, { axiosConfig: { headers }});
  client.use(pluginFetch({ credentials: "include", customFetch: fetch }));
  return client;
}

// SSR context
PersonApi({ fetchFn: fetch }).getPersonById({...});

// CSR context
PersonApi().getPersonById({...});
```

**Comment by smblee** [comment]
Created a suggestion PR https://github.com/ecyrbe/zodios-plugins/pull/203

**Comment by ecyrbe** [comment]
thank you for the PR, it's now available in `v10.6.0` of zodios plugins
